packages/react-sdk/src/core/contexts/MediaDevicesContext.tsx

```diff
@@ -95,13 +97,14 @@ export type MediaDevicesContextAPI = {
   publishAudioStream: () => Promise<void>;
   stopPublishingAudio: () => void;
   stopPublishingVideo: () => void;
+  setInitialAudioEnabled: (enabled: boolean) => void;
   setInitialVideoState: (state: DeviceState) => void;
   selectedAudioInputDeviceId?: string;
   selectedAudioOutputDeviceId?: string;
   selectedVideoDeviceId?: string;
   switchDevice: (kind: MediaDeviceKind, deviceId?: string) => void;
-  toggleAudioMuteState: () => void;
-  toggleVideoMuteState: () => void;
+  toggleInitialAudioMuteState: () => void;
+  toggleInitialVideoMuteState: () => void;
```

Comment:
These are public api methods. Why do they have an `initial` in their name when they are used for regular toggling?

Reply:
Ok, I missed that part. I keep forgetting that both buttons (preview and in-call) live in the same file in the SDK.